airbyte-webapp/src/components/Spinner/Spinner.tsx

```diff
@@ -5,10 +5,11 @@ import styles from "./Spinner.module.scss";
 
 interface SpinnerProps {
   small?: boolean;
+  className?: string;
```

Comment:
Added this so that the Spinner component's styles can be customized by the component that is using it, since I wanted an even smaller spinner size for the `Load more` button.

This approach felt like the most flexible and cleanest change to me, as opposed to trying to add some more custom styles to this component such as an `extraSmall` style.